.fixtures.yml

```diff
@@ -0,0 +1,9 @@
+fixtures:
+    symlinks:
+      tea:  "#{source_dir}"
+      test: "#{source_dir}/spec/fixtures/test"
+    repositories:
+      stdlib:
+        repo: https://github.com/puppetlabs/puppetlabs-stdlib.git
+        ref: 4.12.0
```

Comment:
can you drop the ref please? We try to always test against master, to early detect bugs.